e2e/pages/events/manageOrders.page.js

```diff
@@ -80,6 +91,15 @@ const selectOperation = async (operationType) => {
   await I.runAccessibilityTest();
 };
 
+const selectRelatedToHearing = async (answer) => {
```

Comment:
one of these doesn't need to be async, the other does if it has the accessibility testing run in it

Reply:
👍